src/puppetlabs/jdbc_util/pglogical.clj

```diff
@@ -18,13 +19,16 @@
   pglogical.replicate_ddl_command, escaping quotes and wrapping the statement so
   it won't return anything."
   [sql schema]
-  (str "do 'begin perform "
-       (unsafe-escape-sql-quotes
-        (str "pglogical.replicate_ddl_command('"
-             "set local search_path to " schema "; "
-             (unsafe-escape-sql-quotes sql)
-             "');"))
-       " end;';"))
+  (let [effective-sql (if (sequential? sql)
```

Comment:
aren't strings sequential? Maybe I'm misremembering.

Reply:
I suppose the test below says that it's not

Reply:
sequential? checks to see if the item implements Sequential.  https://clojuredocs.org/clojure.core/sequential_q